setup.md

```diff
@@ -0,0 +1,170 @@
+
+3. Go to your GUAC directory (the rest of the steps need to be done from this directory):
+  ```bash
+  cd guac
+  ```
+
+## Step 2: Build the containers
+
+From your GUAC directory, run:
+
+  ```bash
+  make container
+  ```
+
+## Step 3: Start the GUAC server
+
+  1. In another terminal, from your GUAC directory, run:
+
+  ```bash
+  docker-compose up
+  ```
+  2. Verify that GUAC is running:
+  ```bash
+  docker compose ls --filter "name=guac"
+  ```
+
+  You should see:
+
+  ```bash
+  NAME                STATUS              CONFIG FILES
+  guac                running(7)          /Users/lumb/go/src/github.com/guacsec/guac/docker-compose.yml
+  ```
+
+  **If you don’t see the above,** run `docker-compose down` and try starting up GUAC again. Because Docker Compose caches the containers used, the unclean state can cause issues.
+
+### GUAC Ports
+
+|---
+| Port Number | GUAC Component | Note
+|-|:-|:-:|-:
+| 8080 | GraphQL server | To see the GraphQL playground, <br />visit [http://localhost:8080](http://localhost:8080)  
+|---
+| 4222 | Nats | This is where any collectors that you run will need to <br />connect to push any docs they find. <br />The GUAC collector command defaults to `nats://127.0.0.1:4222` <br />for the Nats address, so this will work automatically    
+|===
+
+## Step 4: Start Ingesting Data
+
+Now you can run the `guacone collect files` ingestion command to load data into
+your GUAC deployment. For example we can ingest the sample `guac-data` data.
+However, you may ingest what you wish to here instead.
+
+```bash
+pushd ../guac-data/docs
+docker run --rm -v $PWD:/data --network guac_default local-organic-guac:latest /opt/guac/guacone collect files /data --gql-endpoint http://guac-graphql:8080/query
+popd
+```
+
+Switch back to the compose window and you will soon see that the OSV certifier
+recognized the new packages and is looking up vulnerability information for
+them.
+
+This command uses the `guacone` GUAC binary from the `local-organic-guac`
+container we built earlier. Because containers don't have access to localhost by
+default, we connect to the existing compose network (`--network guac_default`)
+and use the `guac-graphql` container name to connect to it. The command uses a
+volume mount to mout the `guac-data` into the container for collecting
+(`-v $PWD:/data`).
+
+Alternatively, you may build the GUAC binaries for your local machine and run
+them natively.
+
+```bash
+make build
+./bin/guacone collect files ../guac-data/docs
+```
+
+## Step 5: Check that everything is ingesting and running
+
+Run:
+
+```bash
+curl 'http://localhost:8080/query' -s -X POST -H 'content-type: application/json' \
+  --data '{
+    "query": "{ packages(pkgSpec: {}) { type } }"
+  }' | jq
+```
+
+You should see the types of all the packages ingested
+
+```json
+{
+  "data": {
+    "packages": [
+      {
+        "type": "oci"
+      },
+...
+```
+Congratulations, you are now running a full GUAC deployment!
+## What is running?
```

Comment:
Should this be nested under "Check that everything is ingesting and running"?

Reply:
Updated in my latest branch.